lms/djangoapps/courseware/tests/test_video_handlers.py

```diff
@@ -129,6 +148,8 @@ def test_handle_ajax(self):
             {'speed': 2.0},
             {'saved_video_position': "00:00:10"},
             {'transcript_language': 'uk'},
+            {'do_not_show_again_bumper': True},
+            {'date_last_view_bumper': True},
```

Comment:
Oh, this is data to POST. Personally I would have gone with `bumber_update_last_view_date` for the key that is POSTed.